### PR TITLE
Support for Events as Markdown

### PR DESCRIPTION
**Pull request by tylersmalley**
Just like the blog, events are now managed as Markdown (MDX) files, but under `/data/events`. They will have their own page so instead of just linking off to the event site, we can provide additional content and follow-up.

### CODE REVIEW

File: pages/events/[...slug].tsx
Comment (Xe) [inline pages/events/[...slug].tsx:52]: -Wpedantic, puntable until after merge: I've seen this in a few places, may be worth making it a component? Rule of threes and all.

File: scripts/sitemap.ts
Comment (Xe) [inline scripts/sitemap.ts:1]:
```suggestion
import { generateSitemap } from '@/lib/utils/generate-sitemap.js';
```
Shouldn't it be this?

**Review by Xe — APPROVED** [review]
LGTM, I don't need to re-review if my minor nitpicks are fixed.

File: scripts/sitemap.ts
Comment (tylersmalley) [reply scripts/sitemap.ts:1]: Yup, it can now! It was previously using `esbuild-register` which, for some reason I didn't understand wasn't picking up the path mappings.

**Pull request merged**
